higher_health/views.py

```diff
@@ -1,25 +1,36 @@
 from functools import partial
 
+import pyotp
+from django.contrib.auth import login
+from django.contrib.auth.decorators import login_required, user_passes_test
+from django.contrib.auth.mixins import LoginRequiredMixin, UserPassesTestMixin
+from django.contrib.auth.models import User
 from django.http import HttpResponseRedirect
 from django.shortcuts import render
 from django.urls import reverse_lazy
 from django.views import generic
 
-from .forms import HealthCheckLogin, HealthCheckQuestionnaire
+from .forms import HealthCheckLogin, HealthCheckOTP, HealthCheckQuestionnaire
 from .models import Campus, Covid19Triage, University
 from .utils import get_risk_level, save_data
 
 
-class HealthCheckQuestionnaireView(generic.FormView):
+class HealthCheckQuestionnaireView(
+    LoginRequiredMixin, UserPassesTestMixin, generic.FormView
```

Comment:
Only allow logged in users. Also exclude `is_staff` users as their usernames are likely to not be the `msisdn` and should not be able to complete a health check.